third_party/amd/lib/TritonAMDGPUTransforms/OptimizeEpilogue.cpp

```diff
@@ -73,11 +73,12 @@ static triton::StoreOp convertMfmaLayoutForCDNA4(PatternRewriter &rewriter,
   // order to enable wide 128-bit global stores.
   std::optional<triton::LinearLayout> mfma8Layout =
       triton::gpu::chooseMfmaLikeStoreLayout(valType);
-
-  if (!mfma8Layout)
+  if (!mfma8Layout) {
```

Comment:
While we are here can we rename `mfma8Layout` to `storeLayout` to avoid confusion?